### PR TITLE
Release 1.10.0 in master

### PR DESCRIPTION
**Pull request by mikestephen**
### Proposed changes
Describe the use case and detail of the change. If this PR addresses an issue on GitHub, make sure to include a link to that issue here in this description (not in the title of the PR).

### Checklist
Before creating a PR, run through this checklist and mark each as complete.

- [x] I have read the [CONTRIBUTING](https://github.com/nginxinc/kubernetes-ingress/blob/master/CONTRIBUTING.md) doc
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked that all unit tests pass after adding my changes
- [x] I have updated necessary documentation
- [x] I have rebased my branch onto master
- [x] I will ensure my PR is targeting the master branch and pulling from my branch from my own fork

### CODE REVIEW

**Review by lucacome — CHANGES_REQUESTED** [review]
I think we need to update https://github.com/nginxinc/kubernetes-ingress/blob/master/docs-web/technical-specifications.md#L30 too

**Review by pleshakov — APPROVED** [review]
looks good to me, provided that the changelogs will be updated to match the finalized changelogs in the release PR

re @lucacome comments, putting edge versions as tags would work there

File: docs-web/technical-specifications.md [hunk 1]
```diff
@@ -27,12 +27,12 @@ The supported architecture is x86-64.
       - ``Dockerfile``
       - ``nginx:1.19.6``, which is based on ``debian:buster-slim``
       - 
-      - ``nginx/nginx-ingress:1.9.1``
+      - ``nginx/nginx-ingress:edge``
```
Comment (Dean-Coakley) [inline docs-web/technical-specifications.md:30]: This page should reference the latest production ready release available:
```suggestion
      - ``nginx/nginx-ingress:1.10.0``
```

File: docs-web/technical-specifications.md [hunk 2]
```diff
@@ -27,12 +27,12 @@ The supported architecture is x86-64.
...
     * - Alpine-based image
       - ``DockerfileForAlpine``
       - ``nginx:1.19.6-alpine``, which is based on ``alpine:3.10``
       - 
-      - ``nginx/nginx-ingress:1.9.1-alpine``
+      - ``nginx/nginx-ingress:edge-alpine``
```
Comment (Dean-Coakley) [inline docs-web/technical-specifications.md:35]:
```suggestion
      - ``nginx/nginx-ingress:1.10.0-alpine``
```

File: docs-web/technical-specifications.md [hunk 3]
```diff
@@ -42,7 +42,7 @@ The supported architecture is x86-64.
       - ``openshift/Dockerfile``
       - ``registry.access.redhat.com/ubi8/ubi:8.3``
       - 
-      - ``nginx/nginx-ingress:1.9.1-ubi``
+      - ``nginx/nginx-ingress:edge-ubi``
```
Comment (Dean-Coakley) [inline docs-web/technical-specifications.md:45]:
```suggestion
      - ``nginx/nginx-ingress:1.10.0-ubi``
```

File: docs-web/releases.md [hunk 4]
```diff
@@ -1,5 +1,96 @@
 # Releases
 
+### NGINX Ingress Controller 1.10.0
```
Comment (Dean-Coakley) [inline docs-web/releases.md:3]: For consistency with 1.9.0:

```suggestion
## NGINX Ingress Controller 1.10.0
```

However, I don't see any difference in resulting page: https://deploy-preview-1325--nginx-kubernetes-ingress.netlify.app/nginx-ingress-controller/releases/

**Review by Dean-Coakley — APPROVED** [review]

**Review by lucacome — APPROVED** [review]

**Pull request merged**